index.html

```diff
@@ -23,6 +23,9 @@
       <img src="images/Union.png" alt="">
     </nav>
     <nav class="desktop-nav-menu">
```

Comment:
- [ ] The nav container in the mobile version has to cover all the height of the screen. 
![Captura de pantalla_2021-10-04_11-39-34](https://user-images.githubusercontent.com/53111464/135890560-e7d6510b-ba38-4ba6-b24c-42dbb66adf64.png)
- See at the bottom of the screen there is a space that hasn't been covered.

Reply:
Hello @Alexoid1 , 

Thank you a lot for this insightful review 🤓 . I am currently working to ensure this is rectified.

> HappyCoding